wormhole-connect/src/utils/wallet/index.ts

```diff
@@ -4,20 +4,17 @@ import {
   Context,
   SendResult,
   ChainConfig,
-} from '@wormhole-foundation/wormhole-connect-sdk';
-import {
   postVaaSolanaWithRetry,
```

Comment:
We were previously using `postVaaSolanaWithRetry` from `@certusone/wormhole-sdk`, but now we are using a modified version that sets the priority fee from `@wormhole-foundation/wormhole-connect-sdk`.